imbi/endpoints/project_ssm_configuration.py

```diff
@@ -108,19 +108,27 @@ async def get(self, *args, **kwargs) -> None:
         self.send_response(output)
 
     async def post(self, *args, **kwargs) -> None:
+        project_info = await self._get_project_info(kwargs['project_id'])
         google_tokens = await self._get_google_tokens()
         aws_session = aioboto3.Session()
         body = self.get_request_body()
-        for param in body['values']:
+
+        ssm_path_prefix = self.path_prefix(
+            self.application.settings['project_configuration']
+            ['ssm_prefix_template'], project_info['project_slug'],
+            project_info['project_type_slug'], project_info['namespace_slug'])
+        name = ssm_path_prefix + body['name']
```

Comment:
I'm a little worried about getting slashes wrong here especially since the data is coming from multiple places (eg, SSM params from config file (ultimately from SSM), other config in the database, and stuff from the body). What do you think about changing the `path_prefix()` method to look up `self.settings['project_configuration']['ssm_prefix_template']`, replace the templated params, and also ensure that the result does not end in a slash. This would also clean up the stuff from the previous comment so that the code would look something like:
```python
        ssm_path_prefix = self.path_prefix(
            project_info['project_slug'], project_info['project_type_slug'],
            project_info['namespace_slug'])
        name = f'{ssm_path_prefix}/{body["name"]}'
```

Reply:
Right now it expects the ssm_prefix_template in settings to contain a trailing slash. I'm not sure what we get by updating `path_prefix` to potentially remove a trailing slash only to re-add it back later. And the frontend also expects it to be there.

I'm not worried, but if we were to do something like this I would rather it be directly in `server.py` when it populates the setting, and either we add a trailing slash if it doesn't already exist, or truncate them all off, so either way everyone downstream knows what they're dealing with.

Reply:
Sounds good ... I might go and clean up the expectations a little in server.py if I remember to. What in the UI is expecting it to be there? and, _more importantly_, what are the ramifications of it not being there should someone forget to configure it?

Reply:
It's displayed in the text prefix that's right in front of the text field, so just visual (although this would also probably make someone notice it was wrong). However if you omit it in the config, Imbi will go right ahead and make params with it omitted. So if we do make a change, I think I'm in favor of letting users include or omit it, and if they omit it we add it back in server.py